aws-ts-static-website/index.ts

```diff
@@ -66,7 +63,7 @@ crawlDirectory(
                 key: relativeFilePath,
```

Comment:
Should this use BucketObjectV2?

Reply:
Similar to other comments, trying to avoid moving this to avoid the move back later on in aws V7. BucketObject and BucketObjectV2 are really really close.